Lib/test/test_inspect/inspect_fodder3.py

```diff
@@ -0,0 +1,12 @@
+from functools import cached_property
+
+class Parent:
+    @cached_property
+    def foo(self):
+        "this is the docstring for foo"
+        pass
+
+class Child(Parent):
+    @cached_property
+    def foo(self):
+        pass
```

Comment:
```suggestion
from functools import cached_property

class Parent:
    @cached_property
    def foo(self):
        """this is the docstring for foo"""

class Child(Parent):
    @cached_property
    def foo(self):
        pass
```

Can we also have a test where the parent cached property has no docstring but the child does?